docs/ssss.md

```diff
@@ -1,3 +1,94 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that runs on a
+blockchain, governed by rules set in a contract, and operates autonomously
+without any government backing.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
+
+The most common way to implement a DAO is by **using a mainnet that supports
+smart contracts**. You can code your governance rules in a contract and deploy
+it on the mainnet, which will interact with the mainnet's ecosystem. However,
+this approach **has some disadvantages**:
+
+1. It requires payment of a gas fee.
+2. It is dependent on the chosen mainnet's performance, security, reliability,
+   usability, political risks, etc.
+3. It is limited to the ecosystem of the chosen mainnet and cannot easily
+   interact with other mainnets.
+
+Simperby solves these problems by providing a standalone blockchain for a DAO.
+
+For more details, refer to [the comparisons (WIP)](./comparison.md)
+
+## Simperby
+
+Simperby is a blockchain engine that builds a standalone blockchain for a DAO.
+
+- Trustless
+- Distributed
+- Fault Tolerant
+- Lightweight
+- Highly Interoperable
+- Native Support for Multichain Treasuries
+
+For details on the core protocol, refer to the [protocol
+overview](./protocol_overview.md).
+
+### A Byzantine Fault Tolerant System
+
+- **Builds an independent layer-1 mainnet** like [Cosmos
```

Comment:
I think it's little weird if all sentences are not in the consistent form. Some sentences have omitted subjects (which would be Simperby)
> Builds an independent layer-1 mainnet

while some are not
>  A peer-to-peer voting mechanism (governance) is used to make decisions on the
 organization, which is then finalized by the consensus.

and some are not even full sentences
> One mainnet for one DAO
> No tokens, no staking, no mining, no gas fee...

I'm not sure what would be the best way to rewrite them, but the current one seems a little disorganized for me.

Reply:
I'm not sure too. I think it might not be necessary to keep them consistent as they could be
- a property
- a feature that it has
- a point worth mentioning
- a phenomenon happens upon it

Reply:
Let's keep it as is for now